config/locales/client.en.yml

```diff
@@ -187,6 +187,7 @@ en:
 
     themes:
       default_description: "Default"
+      broken_theme_alert: "⚠️ Theme/component %{theme} has errors which may prevent your site from working correctly! You can disable it at %{path}."
```

Comment:
After this PR gets merged, please watch the [Timeline at Transifex](https://www.transifex.com/discourse/discourse-org/settings/timeline/) carefully for "Autofetch failed" errors. A year ago an Emoji stopped Transifex from doing auto updates. It will be interesting to see if they fixed the bug in the meantime. I highly doubt that it will work, so be prepared to show the warning sign in another way.

Reply:
Thanks @gschlager for the heads up. I did notice there is another string with an emoji here:

https://github.com/discourse/discourse/blob/1fbe078ae074f29395d79db0856d1991f1675ab5/config/locales/client.en.yml#L182

This line was last changed 2 years ago, so maybe the problem is specific to a set of emojis? This makes me wonder if emojis have to "translatable"?

Reply:
Yes, I think it depends on the Emoji. Transifex expects some of them to be represented as Unicode Surrogates which isn't supported by Ruby. You might not even need to include this Emoji in the translation as long as it still works with RTL. There's definitely no need to translate a warning sign.

Reply:
Ok I've moved the emoji from `client.en.yml` to the JS file and tested with an RTL locale and it looked alright 👍 

Will merge this PR once the build passes.